### DISCUSSION TITLE
Molten Core Tuning Feedback

### DISCUSSION THREAD

**Discussion by Annamaria-CC** [chromiecraft/chromiecraft#2439] [PTR Feedback]
~~In this discussion you can give us feedback regarding the Molten Core Tuning that is currently active on the PTR. This tuning will change over time and is far from finished.~~

You can now report feedback on the tuning for the live version of Molten Core on ChromieCraft.
 This is not for bug reports. You can report them [here](https://github.com/chromiecraft/chromiecraft/issues).

**Comment by kimbowned** [comment]
Hi, i did a ptr run 10ish days ago with a group and it seemed way over the top. Are you guys planning to nerf it down for release or is it staying as it is currently. Because the straight damage/healing reduction is way to much and it takes the fun out of healing and dps since you basically dps/heal like ur lvl 25 again. I wouldnt mind if you buffed bosses hp/armor/ressistance values cuz that way at least ur able to see big numbers. Considering its the first raid in wow and you've basically made it a 20man raid instead of 40 is all this extra nerf to output really nesessery? Also considering that 90%(is not more) of the player base from what i can tell is casual this could be a real turnoff. I understand if you made it like an optional mode(like hc) for those who wants to be competative and challenge themselves but for most i think its going to be a little to much. This is coming from a guy who played classic and multiple vanilla pservers before that and playing in semi to straight hc guild.

**Comment by balleny** [comment]
Is only heal affected by the reduction or also absorbs? otherwise, you will create plenty of disbalance

**Reply by Annamaria-CC** [reply]
both :)

**Comment by mgorki123** [comment]
WYR from Chromie-homies here and I will share my opinion with you guys on the tuning. In general I really liked the overall tuning of the raid because I believe it achieved what you were aiming for: Easy in the beginning but a decent challenge towards the end. The first 8 bosses were not challenging but also not too easy that you can just do whatever you want, you can still manage to wipe to Garr or Geddon if you try hard enough, you still need to know how to tackle them properly. But once any guild or pug has a couple tries on them they will not pose any problem whatsoever. And this is a good thing, the gear acquired through those bosses make it easier and easier to tackle Ragnaros every week. People can also do a lot of trash runs or buy materials from the AH and craft really strong gear, especially for their tanks, which will make their lives significantly easier.  This gives a good sense of progression and accomplishment. As for Majordomo and Ragnaros, I think they are tuned just right. Not too hard that no one will clear it without hundreds of wipes, and not easy and requires a good amount of effort both in preperation and in execution (guild coordination)

**Comment by dumikachromiecraft** [comment]
I can only echo Wyr's sentiment. You don't _need_ every player to clear it on week 1. The same thing happened with UBRS, people called it hard, but 3 weeks later everyone was farming it on a regular basis. The crafted gear acquired from the trashes and the high ilvl gear dropped from bosses will make Ragnaros a _lot_ easier for everyone. Originally, back in 2005, Ragnaros was cleared a whopping 2 month after Majordomo was killed.

In my opinion, MC is fine as it is. If you make it easier, people will cut through it like knife on butter, and we won't get to test the bugs that may arise.

**Comment by RestoCC** [comment]
Resto from Synergy here. Our guild has an overwhelmingly positive experience in clearing MC on PTR and now in the beta version upon release.

**Trash Tuning**
After the trash was scaled back in an earlier build on PTR, I can say that the tuning for it is perfect. We know that for most guilds they will need to do some MC trash clears and this decision allows all guilds to do so without fear of hitting a brick wall. Evidence of this can be seen in /world with PuG groups and Guilds LFM for MC runs less than 24hours after release. For guilds that have the utility and manpower, you can pull 2-4 groups of dogs before Magmadar, or 2-4 Ancient Core Hounds and have pallies bubble off the stacks to effectivly nullify the debuffs on the tanks but for alot of guilds who may not be able to do that, the tuning is spot on for holding 1 and maybe 2 at once.

**Aggro radius on Trash**
I would like to preface by saying we as a guild (and possibly others?) have zero context on the aggro radius of the MC trash in a wotlk client versus Classic but we can say with certainty that the aggro radius is much smaller and therefore easier to abuse than in Classic. We tested extensively on PTR and knew we could skip an insane amount of mobs. An example of this is the first 5minutes of the run as we headed towards Lucifron, with the exception of needing to do the MC quest, we could skip nearly every pack before the bridge versus some other guilds that opted to clear extra.

**Boss Tuning**
Tuning was on point from Lucifron to Ragnaros. The first few bosses are a fantastic entry level point for guilds that may not have the experience of veteran players or those that can commit hours to their characters. They still provide challenges but can be overcome with simple yet effective strategies as well as bringing particular classes if needed. Examples are having a good number of decurses for Lucifron- simple decurse mechanic but can prove detrimental. I think Shazzrah was a fantastic tuned fight. Domo was an exceptionally tuned fight. We always planned to bring enough mages to nullify the fight but for the guild that may progress for the next few weeks raiding once a week, this fight and the damage from the Elites aoe + Shadow Bolts (if healers cannot be sheeped) i think is a great test for guilds as a precurser to Ragnaros.

**Ragnaros**
The tuning of this fight surprised us but after the kill we can say with certainty that the developers got this correct as well. I look forward to guilds that progress on this fight in the coming weeks and seeing their reactions and elation to a kill because it will be well-deserved. The damage with Wrath of Rag seems as well as the direct damage to tanks are the keys to victory here. P2 can be nullified with 1-2 hunters but from my experience in Classic in the rare occasions we ever saw a P2 it was never a challenge. Ensuring tanks can survive the hits from Ragnaros will be key. I dont know if the devs plan to make any future tuning changes but I think that it should be left for now at least.

**Closing thoughts**
- Tuning across the board is fantastic
- I expect many guilds to struggle with Sulfuron/Domo/Rangaros in the coming weeks but those guilds should persevere and perfect their strategies and perhaps gear rather than hop for future nerfs
- If there are sources for the aggro radius of trash out there i would be interested in seeing anything as we cannot confirm its incorrect and that its just our opinion compared to a different client.
- The Devs who worked on this for 9months should be incredibly content and proud of themselves

**Comment by balleny** [comment]
On the other hand I am wondering if there is yet a Ragnaros kill, where a non-tank melee survived until the end. This might be something to consider. I would call it well fixed and tuned, when I see a 5 melee dps group survive ;)

**Comment by Jonny727272** [comment]
Fondle from The Tribunal here.
We made it 9/10 in MC and will try to get some more Rag attempts in tonight.

As others have said, the first 8 bosses are tuned really well and work fine. There are def cases where we feel the struggle of only having half the people the raid was meant for. Getting decurses and purging magic debuffs is a struggle at times with just how many are going out. I think we had at least one wipe on every boss, except for Golemag. 

But moving onto the final two bosses, I do have some input. Domo's healers should be able to be CC'ed by other spells. That could be repentance or druid's sleep or a warlock fear. We were lucky to have 3 mages to sheep, but then had a warlock use their felguard to tank the final healer. It was a silly strat, but it worked. The original plan was to kill the healers first, then elites, but the elites proved to do so much AOE damage, that we killed all of them first. Since we were able to down Domo after 4 or 5 attempts, I'm not sure it needs to be tuned, but it's worth keeping an eye on it as other guilds try to get into MC. 

Since we only had 2 attempts on Rag, I don't feel I can give any real good input. I will comment that that explosions on casters seemed to be doing a lot more damage than what I was expecting. I saw a couple pop for 7k, which no caster will be prepared to live through. We can't have them wear too much FR gear as they still need to work on DPSing Rag. The other concern is just with melee being pretty much useless as they die so fast. I'm not sure what can be done about this, but it isn't fun or challenging. Excluding an entire type of damage dealer isn't what I would call fun game design, and just makes the whole encounter a bit more annoying since the HP pool of Rag is already so dang high.

My biggest wish for these vanilla raids, would be to allow 25 players to enter the raid. We will have 25man raids in TBC and WOTLK, so why not start that now? This would be a pretty big buff to the players, so maybe wait a couple weeks first.

All in all, I think this is a pretty good first raid tier for the server. It's fun, well tuned, and we feel a sense of progression after each boss fight. Thanks CC staff.

**Comment by Brylezor** [comment]
I only cleared once up to domo, mostly it seems with some optimization on tactics the tuning is fine. 

I do not support the reducing players healing or damage method of making the raids harder, I think increasing the health and damage numbers of the bosses and mobs are more then enough, and doing anything more then that is just changing game mechanics too much that it is invalidating the testing portion of this server, changing a fight difficulty because now you want an extra healer to counter something that is not an original design (healing reduction) defeat the purpose of testing the server the way its meant to be.

That is just my opinion though, and like I said I was against the damage numbers reduction in UBRS as well, it defeats the purpose of mmorpg in the way that ur character feels like it got weaker going into a raid, which is the final testing ground for gear and performance progression, hopefully this wont be the case in future raids.

my second point is more of a fear that as a melee class main (enh shaman) i will walk into Rag fight and be quickly replaced by a ranged dps because the damage to melees is unhealable (that is what most people who cleared MC are claiming) and it is much more profitable to bring a range dps over a melee dps, which is a reoccurring theme in wow history fights, when fights are hard- range dps is better more often then not.
i hope this will be addressed, but I will reserve further thoughts after i have seen rag encounter for myself.

**Reply by RestoCC** [reply]
Hey mate melee should just put on some fire resistant gear- having 140 should be adequate and if you don’t want to sacrifice too much gear you can get at least 60 from enchants alone but I’ll leave the specifics to you as I’m not an expert on your class. You should not guaranteed to be benched for the fight just because of being a melee but don’t get that confused with the truth that range have an easier time and are therefore favoured. It swings both ways though, there will be times range may not be optimal even though historically in wow range does win. It’s the nature of choosing melee.

You said you felt the tuning was fine but also said that you think the healing dampen should not be there? Not disregarding your opinion but did you happen to be on the PTR before they made these changes? The blanket hp and damage buffs they originally had was across the entire instance resulting in trash taking far longer than it should and creating a tedious chore.

I agree that players may fee unsatisfied that their characters can feel weaker but if you look at it from a more positive view, perhaps you could view it as a challenge on content that is already quite old and repetitive. Originally I was against any tuning and the 20man but it’s been a blast since then and made it easier to do split runs and even show fresh people the content.

Let me know your thoughts

**Reply by Brylezor** [reply]
cleared rag as enh shaman, with few attempts as resto shaman ( mostly due to tanks going into the floor with charge causing wipes)

the tuning is fine. but as i mentioned i am very against the tuning the player power to add difficulty, in ubrs i did it once and refused to do it again until it was changed, it was just frustrating to dps and get those numbers. same with healing now in MC, since i cleared it now as enh, i will probably not gonna go healing in MC so its fine for me, but when i did have to go healer in MC it felt very frustrating.

I would rather do 10 man MC with buffs to boss/mobs where players damage/healing/health just remains true to the numbers out of raid.

I do understand that is just my opinion though, but I came to this server not because I was looking for a challenge, it was just for fun and maybe helping the core with bugs and issues, only to find the server constantly trying to make it challenging for some reason (guessing people request for it or something) 


on tuning point- 

tuning is fine for 20 man we have 2 groups and one group full cleared MC (more experienced wow guys) the other group was slugging through it a bit longer but eventually managed to keep progressing (currently downed baron) 
as someone who recently played classic wow i can say that fights are a lot harder then in classic, but mostly because classic was a breeze.

major issues-
majordomo fight - we had one hunter kiting an Elite, he took him far and i think he died and caused the elite to start evading, we thought it broke the fight but we had the 2nd on threat for that add to move down and when he did the Elite targeted him and returned to the fight.  it was awkward but it worked for us.  
another thing was the fact that one of the adds dropped the leaf for hunter quest, i thought it was suppose to be a drop from the chest and not the adds, i might be wrong.

Raggy fight-
at first attempt on initial summon, during the RP scene of him coming up the melee group who were positioned behind the boss (where majordomo stands) was taking a lot of damage while raggy was not even targetable, this caused a lot of confusion and eventually 3/5 melees died before boss became targetable.

and the other big issue is charge pathing, this happened multiple times where the tanks got knocked up, charged back into raggy and went straight through the floor, causing a wipe as they did not even die just start falling into the world.
This consistently happened on every pull, even in the pull we got the kill but lucky the other tank managed to keep things going and did not fall into the world.

Sorry for the long text.

**Reply by Jonny727272** [reply]
Some tips for Rag and avoiding the bugs. We had all pet classes dismiss their pets before starting RP and that seemed to fix the damage coming in before the encounter starts, And we also had our warrior/bear tank wait until they were on the ground to charge and that seemed to work fine too.

**Reply by Annamaria-CC** [reply]
Pretty sure we have a fix live for the pet issue on rag.

**Reply by Jonny727272** [reply]
Oh, we'll give it a try this week then. It is really only an issue on the first pull as that is when the RP event is. Everything after that is a mute point as Rag is already there.

Thanks for the heads up.

**Comment by UnicornManure** [comment]
Hey i think mostly the raid tuning has been on point atleast for the first 8 bosses. It requires everyone to be active & engaged in the fight, no semi afk players like in classic can get dragged through it. Sure you can at times feel like the raid was made for 40 players but its still doable with  20 players even for the casual guilds.

The trash is at a good level, any 10 man can just come in and farm them.

The only thing i think is wrong is the majordomo fight its to dependent on mages since they are the only class that can cc the healers. For everthing else there seems to be more than one class that can handle it. For Magmadars fear warrior, priest & shamans have a counter to it, for decurses you got druids & mages, for magic dispel paladins & priests. Sure only warlocks can banish garr adds but the fight is still managable without it. Heck even majordomos elites are kiteble by atleast mages & hunters. I think you should add so alteast 1 more class can use thier cc mechanics on the healers doesnt matter if its fear, repentance, sleep or a freeze trap.

Right now it feels very punishable for a small casual guild if your mages can't make it or you cant get any, and you shouldnt need to have a mage alt backup just for that fight.
